src/flask_track_usage/storage/__init__.py

```diff
@@ -75,7 +98,13 @@ def __call__(self, data):
         :Parameters:
            - `data`: Data to store.
         """
-        return self.store(data)
+        result = self.store(data)
+        data["_parent_class_name"] = self.__class__.__name__
```

Comment:
Since ``result`` is being used, but data isn't, can these assignments be dropped?

Reply:
I'll dive into that. Don't recall my original reasoning. The summary classes are reliant on the `_parent_class*` assignments. I suspect that `data` and `result` are, in the end, the same dictionary reference; so I've got that very confusingly written. I might be able to remove the `result` variable. Will check on it.